### PR TITLE
Include rollout dashboard, borrowed from Mimir

### PR DESCRIPTION
**Pull request by zalegrala**
**What this PR does**:

* Include rollout dashboard
* Adjust Grafana label to `tempo-mixin`

**Checklist**
- [ ] Tests updated
- [ ] Documentation added
- [x] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: operations/tempo-mixin/dashboards/rollout-progress.libsonnet [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+          id: 1,
+          gridPos: { h: 8, w: 10, x: 0, y: 0 },
+        },
+
+        //
+        // Writes
+        //
+        $.panel('Writes - 2xx') +
+        $.newStatPanel(|||
+          sum(rate(cortex_request_duration_seconds_count{%(gateway_job_matcher)s, route=~"%(gateway_write_routes_regex)s",status_code=~"2.+"}[$__rate_interval])) /
```
Comment (joe-elliott) [inline operations/tempo-mixin/dashboards/rollout-progress.libsonnet:96]: I think all these `cortex_` metrics should be `tempo_` metrics? Double check by importing the dashboard into an env that has tempo to make sure these graphs are populated.

File: operations/tempo-mixin-compiled/dashboards/tempo-reads.json [hunk 2]
```diff
@@ -1498,7 +1498,7 @@
  "schemaVersion": 14,
  "style": "dark",
  "tags": [
-  "tempo"
+  "tempo-mixin"
```
Comment (annanay25) [inline operations/tempo-mixin-compiled/dashboards/tempo-reads.json:1501]: I would prefer to keep this "tempo" as it shows up nicely in the Grafana Dashboard explore. 
image

but no strong opinion!

**Review by annanay25 — APPROVED** [review]
Nice one Zach 🚀

File: operations/tempo-mixin-compiled/dashboards/tempo-reads.json (hunk 2 above)
Comment (zalegrala) [reply operations/tempo-mixin-compiled/dashboards/tempo-reads.json:1501]: Once nice thing about changing it is that we'd be able to tell that those dashboards came from the mixin.  I was thinking about this in the context when other dashboards get created that aren't part of the mixin, knowing where to make changes requires some head scratching.

**Comment by zalegrala** [comment]
image

**Pull request merged**